tensorflow/lite/micro/kernels/micro_tensor_utils.h

```diff
@@ -42,833 +43,16 @@ class CpuBackendContext;
 
 namespace micro_tensor_utils {
```

Comment:
let's use a single tflite namespace. This will mean changing the function names.